Branch-SDK/Branch-SDK/BNCLinkData.h

```diff
@@ -21,6 +21,7 @@
 @property (nonatomic, strong) NSString *stage;
 @property (nonatomic, strong) NSString *params;
 @property (nonatomic, assign) NSUInteger duration;
+@property (nonatomic, assign) NSString *ignoreFirstClick;
```

Comment:
Not sure why this was changed to a string; if you're passing user agent here instead, then you should probably rename this to an entirely new variable.